lib/data/build-data.ts

```diff
@@ -58,6 +60,8 @@ export class AndroidBuildData extends BuildData {
 		this.keyStoreAliasPassword = data.keyStoreAliasPassword;
 		this.keyStorePassword = data.keyStorePassword;
 		this.androidBundle = data.androidBundle || data.aab;
+		this.buildFilterDevicesArch = !this.androidBundle && data.filterDevicesArch !== false ;
+		this.gradleFlavor = data.flavor;
```

Comment:
Should this be named `data.gradleFlavor` for consistency?

Reply:
I can yes. But to me `flavor` is more of an android option than a gradle option. I mean `gradlePath` and `gradleArgs` are options for the gradle "tool". 
Let me know what you prefer and i ll change